harness/determined/deploy/gke/cli.py

```diff
@@ -0,0 +1,551 @@
+            + "set-cluster <other_cluster_name>`\nto interact with other deployed clusters."
+        ).format(args.cluster_name),
+        "yellow",
+    )
+    cmd = ["kubectl", "config", "set-cluster", args.cluster_name]
+    subprocess.check_call(cmd)
+    cmd = ["helm", "uninstall", "determined-gke"]
+    subprocess.check_call(cmd)
+    cmd = [
+        "gcloud",
+        "container",
+        "clusters",
+        "delete",
+        args.cluster_name,
+        "--region",
+        args.region,
+        "--quiet",
+    ]
+    subprocess.check_call(cmd)
+    print("Succesfully deleted GKE Cluster {}".format(args.cluster_name))
+
+
+args_description = Cmd(
+    "gke",
+    None,
+    "GKE help",
+    [
+        Cmd(
+            "up",
+            handle_up,
+            "create gke cluster",
+            [
+                ArgGroup(
+                    "required named arguments",
+                    None,
+                    [
+                        Arg(
+                            "--cluster-name",
+                            type=str,
+                            default=None,
+                            required=True,
+                            help="a unique name for the gke cluster",
+                        ),
+                        Arg(
+                            "--gpu-node-pool-name",
+                            "--gpu-name",
+                            type=str,
+                            default=None,
+                            required=True,
+                            help="a unique name for the GPU node pool",
+                        ),
+                        Arg(
+                            "--gcs-bucket",
+                            type=str,
+                            default=None,
+                            required=True,
+                            help="a unique name for the GCS bucket that will store your"
+                            " checkpoints",
+                        ),
+                    ],
+                ),
+                ArgGroup(
+                    "optional named arguments",
+                    None,
+                    [
+                        Arg(
+                            "--gpu-type",
+                            type=str,
+                            default=defaults.GPU_TYPE,
+                            required=False,
+                            help="accelerator type to use for agents",
+                        ),
+                        Arg(
+                            "--gpus-per-node",
+                            type=int,
+                            default=defaults.GPUS_PER_NODE,
+                            required=False,
+                            help="number of GPUs per node",
+                        ),
+                        Arg(
+                            "--helm-dir",
+                            type=str,
+                            default="helm/charts/determined",
```

Comment:
does it work when your current working directory is not the repo root? 
if not, can we have the helm resources location "automatically discovered" to make it work?